example/android/app/src/androidTest/java/com/example/InstabugSample/util/InstabugViewMatchers.java

```diff
@@ -0,0 +1,16 @@
+package com.example.InstabugSample.util;
+
+import android.graphics.Color;
+import android.view.View;
+
+import org.hamcrest.Matcher;
+
+public class InstabugViewMatchers {
```

Comment:
**suggestion:** Why do we need to wrap the custom matchers in a utility class instead of using them directly? I personally prefer if they were used directly; to avoid this class from having so many unrealted matchers in the future.

Reply:
I did that to keep things consistent with how Espresso does it, to be able to call things like `onView(matcher).check(matches(hasBackgroundColor(color))` instead of `onView(matcher).check(matches(new HasBackgroundColorMatcher(color))